src/spikeinterface/extractors/neoextractors/openephys.py

```diff
@@ -244,6 +244,7 @@ def __init__(
                 experiment_names=experiment_names,
             )
         )
+        self._stream_folder = stream_folder
```

Comment:
Can you move this before that `self._kwargs` logic? We try to keep that at the end of the init so we know where to look for it!

Reply:
Of course :) Good to learn these conventions, thanks for your patience.